CHANGELOG.md

```diff
@@ -44,6 +44,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   function parameter. For example, `author(foo@)` is now an error, and
   a revset alias `'revset-aliases.foo@' = '@'` will be ignored. 
 
+* A default revset-alias function `trunk()` now exists. If you previously defined
+  your own `trunk()` alias it will continue to overwrite the built-in one, but future
+  functionality will assume that the function resolves correctly.
```

Comment:
I'd move everything after "...continue to overwrite the built-in one" to config.md or revsets.md. See also my comment on revsets.md.